verify/go-tools/verify-gometalinter.sh

```diff
@@ -14,10 +14,17 @@
 # limitations under the License.
 
 set -o errexit
-set -o nounset
 set -o pipefail
 
-gometalinter --deadline=180s --vendor \
+DEADLINE=""
+
+if [[ "$GOMETALINTER_DEADLINE" ]]; then
+  DEADLINE="$GOMETALINTER_DEADLINE"
+else
+  DEADLINE=180s
+fi
+
+gometalinter --deadline=$DEADLINE --vendor \
```

Comment:
a simpler fix would be to use
```shell
--deadline="${GOMETALINTER_DEADLINE:-180s}"
```

then you don't need the if statement, and you can keep `set -o nounset`.

Reply:
thank you for your suggestion 👍. I've updated it. PTAL